notebooks/Tutorial8-Time_series.py

```diff
@@ -284,4 +288,85 @@ def plot_series(
 )
 plot.show()
 
-# As for any data, privacy metrics need to be computed on the output time series data to confirm that it is anonymous.
+# ## Privacy metrics for time series
+
+# As for any data, privacy metrics need to be computed on the output time series data to confirm that it is anonymous. We first retrieve the data required by the privacy metrics, that is the original datasets and their sensitive unshuffled counterparts. The unshuffled datasets are only used for computing the privacy metrics
+
+# +
+# we get the unshuffled vanilla avatars from the job result based on the original id
+vanilla_unshuffled_avatars = [
+    result.sensitive_unshuffled_avatars_datasets
+    for result in job.result.datasets
+    if result.original_id == dataset_vanilla.id
+][0]
+
+# we get the unshuffled time series avatars from the job result based on their original ids
+ts_unshuffled_avatars = [
+    result.sensitive_unshuffled_avatars_datasets
+    for result in job.result.datasets
+    if result.original_id != dataset_vanilla.id
+]
+ts_original_ids = [
+    result.original_id
+    for result in job.result.datasets
+    if result.original_id != dataset_vanilla.id
+]
```

Comment:
I always a little bit afraid of doing thuf like this, I feel dangerous to iterate to time , what happens if the list order change? 😱

Reply:
it is OK if the order of `job.result.datasets` changes because we access `result` in the same order to fill both `ts_unshuffled_avatars ` and `ts_original_ids ` and we use only objets from `result`.
Would you prefer using `zip` ?